### PR TITLE
Create build-node-examples.yml

### PR DESCRIPTION
**Pull request by jbolda**

### CODE REVIEW

File: .github/workflows/build-node-examples.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    steps:
+      - name: checkout tauri
+        uses: actions/checkout@v2
+        with:
+          repository: tauri-apps/tauri
+          ref: dev
+      - name: checkout examples
+        uses: actions/checkout@v2
+        with:
+          path: examples
```
Comment (nothingismagick) [inline .github/workflows/build-node-examples.yml]: maybe this is one examples too many

File: .github/workflows/build-node-examples.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          key: ${{ runner.OS }}-build-reg-index-${{ hashFiles('**/Cargo.toml') }}-
+      - name: cache rust registry/cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ format('{0}/registry/cache/', matrix.CARGO_HOME) }}
+          key: ${{ runner.OS }}-build-reg-cache-${{ hashFiles('**/Cargo.toml') }}-
+      - name: build rust
+        run: cargo build
+        env:
+          TAURI_DIST_DIR: ../../test/fixture/dist
```
Comment (nothingismagick) [inline .github/workflows/build-node-examples.yml]: this looks wrong

File: .github/workflows/build-node-examples.yml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        working-directory: ./cli/tauri.js
+        run: yarn
+      - name: cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ${{ format('examples/node/{0}/node_modules', matrix.example.folder) }}
+          key: ${{ runner.OS }}-build-${{ hashFiles(format('examples/node/{0}/yarn.lock', matrix.example.folder)) }}
+          restore-keys: ${{ runner.OS }}-build-${{ env.cache-name }}-
+      - name: install via yarn
+        working-directory: ./examples/node/${{ matrix.example.folder }}
```
Comment (nothingismagick) [inline .github/workflows/build-node-examples.yml]: and here is the third examples...?

File: .github/workflows/build-node-examples.yml (hunk 1 above)
Comment (jbolda) [reply .github/workflows/build-node-examples.yml]: The idea with this is to emulate what we had before. A `tauri` folder with `tauri/examples`. I _think_ this is correct.

File: .github/workflows/build-node-examples.yml (hunk 3 above)
Comment (jbolda) [reply .github/workflows/build-node-examples.yml]: third?

File: .github/workflows/build-node-examples.yml (hunk 2 above)
Comment (jbolda) [reply .github/workflows/build-node-examples.yml]: Hmm, are you thinking this needs another folder level as well?

Comment (jbolda) [reply .github/workflows/build-node-examples.yml]: Changing that doesn't seem to make a difference.
https://github.com/tauri-apps/examples/pull/7/checks?check_run_id=443575019#step:15:10

**Comment by jbolda** [comment]
We are hitting an issue with a path (I believe), and I have traced it through the `js` to here: https://github.com/tauri-apps/tauri/blob/dev/cli/tauri.js/src/runner.ts#L265. The actual error is on the rust side though, and I don't know how to trace that handoff through.

**Comment by jbolda** [comment]
Thanks for the assist @tensor-programming!

**Comment by jbolda** [comment]
and I think we are golden!

**Pull request merged**